tests/test_processes_xy.py

```diff
@@ -128,3 +128,33 @@ def test_power(arg_in: Dict[str, Any], arg_out: str):
     out = map_required(node_id, process_def)
 
     assert out == process_ref
+
+
+@pytest.fixture(
+    params=[
+        ({'data': {'from_node': 'cube'}, 'parameters': (0,1,0.5), 'function': {'from_node': 'udf'}, 'dimension': 't'},
+         "{'data': _cube, 'parameters': (0, 1, 0.5), 'function': _udf, 'dimension': 't'}"),
+        ({'data': {'from_node': 'cube'}, 'parameters': {'from_node': 'data_2'}, 'function': {'from_node': 'udf'}, 'dimension': 't'},
+         "{'data': _cube, 'parameters': _data_2, 'function': _udf, 'dimension': 't'}"),
+        ({'data': {'from_node': 'cube'}, 'parameters': {'from_node': 'data_2'}, 'function': {'from_node': 'udf'}, 'dimension': 't', 'labels': None},
+         "{'data': _cube, 'parameters': _data_2, 'function': _udf, 'dimension': 't', 'labels': None}"),
+        ({'data': {'from_node': 'cube'}, 'parameters': {'from_node': 'data_2'}, 'function': {'from_node': 'udf'}, 'dimension': 't', 'labels': '2000-01-01T00:00:00Z'},
+         "{'data': _cube, 'parameters': _data_2, 'function': _udf, 'dimension': 't', 'labels': '2000-01-01T00:00:00Z'}"),
+        ({'data': {'from_node': 'cube'}, 'parameters': {'from_node': 'data_2'}, 'function': {'from_node': 'udf'}, 'dimension': 't', 'labels': ['2000-01-01T00:00:00Z', '2000-01-02T00:00:00Z']},
+         "{'data': _cube, 'parameters': _data_2, 'function': _udf, 'dimension': 't', 'labels': ['2000-01-01T00:00:00Z', '2000-01-02T00:00:00Z']}"),
+    ]
+)
+def get_fit_param(request):
+    return create_params("node_xy", request.param[0], request.param[1])
```

Comment:
Just for consistency it would be nicer to call the node `node_curve` instead of `node_xy`